src/test/java/seedu/address/logic/parser/ExportCommandParserTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.commands.ExportCommand.MESSAGE_USAGE;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.ExportCommand;
+
+/**
+ * Test cases for export command.
+ */
+public class ExportCommandParserTest {
+
+    private ExportCommandParser parser = new ExportCommandParser();
+
+    @Test
+    public void parse_validArgsEmail_returnsExportCommand() {
+        assertParseSuccess(parser, "export email", new ExportCommand("email"));
```

Comment:
Perhaps can include the capitalized `email` and `phone` to show it is case insensitive?

Reply:
Good catch. Will do it thanks